README.md

```diff
@@ -16,13 +16,14 @@ By default, `kcp` only knows about:
 
 Like vanilla Kubernetes, `kcp` persists these resources in etcd for durable storage.
 
-Any other resources, including Kubernetes-standard resources like [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)s, [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s and the rest, can be added as [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s and optionally reconciled using the standard controllers launched against the kcp API, or via two-way replication into a Kubernetes cluster.
+Any other resources, including standard Kubernetes resources like [`Deployment`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)s and the rest, can be added as [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s (CRDs) and optionally reconciled using the standard controllers launched against the kcp API, or via two-way replication into a Kubernetes cluster.
```

Comment:
The previous sentence about etcd can probably be changed now - I don't know that etcd is an important detail anymore, but we do want to connect the pedigree to Kubernetes and imply that this is refining down the core control plane part (kube-apiserver or w/e) in order to accomplish <reason>.